src/Symfony/Component/WebLink/Link.php

```diff
@@ -175,6 +175,11 @@ public function getRels(): array
         return array_values($this->rel);
     }
 
+    /**
+     * Returns a list of attributes that describe the target URI.
+     *
+     * @return array<string, scalar|\Stringable|list<scalar|\Stringable>>
```

Comment:
please also fix the `@var` of the property, which misses the `\Stringable` type (and some scalar types).

Reply:
The `@var` type was correct according to [LinkInterface](https://github.com/php-fig/link/blob/5b8d54ad01c46baa3bb13388640a62fdb34461a5/src/LinkInterface.php#L49-L51):
> the value is either a PHP primitive or an array of PHP strings

Reply:
Then our implementation is broken, as our setter allows setting `Stringable` but does not cast them to string, which means `getAttributes` may return non-scalar values.

Reply:
and then it means that the new type added here is wrong (there is no reason for the private property and its getter to have different types, as the getter returns the property value as is)

Reply:
I don't think we should change the implementation to cast stringables to strings.
Stringables are useful to create "lazy strings" and casting too early would break this capability.
We could cast on first *read* (and possibly memoize).
BUT: what a mess of complexity to fit one bad idea; strict types namely...

I'm fine keeping as proposed personally, or changing only here (keep the argument) to:
`@return array<string, scalar|list<scalar>>`

Reply:
I think the current type is simple and meets needs. I don't know to silent the psalm error.